build/BuildSteps.cs

```diff
@@ -637,18 +519,6 @@ public static void GenerateSBOMManifestForZips()
             }
         }
 
-        public static void DotnetPublishForNupkg()
```

Comment:
how come we don't need this anymore?

Reply:
the in-proc branch should not be publishing this package. I also found that this package is not being published anymore to the nuget gallery.